packages/ngtools/webpack/src/paths-plugin.ts

```diff
@@ -12,15 +12,15 @@ import {
   NormalModuleFactoryRequest,
 } from './webpack';
 
-
 export function resolveWithPaths(
   request: NormalModuleFactoryRequest,
   callback: Callback<NormalModuleFactoryRequest>,
   compilerOptions: ts.CompilerOptions,
   host: ts.CompilerHost,
   cache?: ts.ModuleResolutionCache,
+  dtsModuleResolutionCache?: Map<string, string>,
 ) {
-  if (!request || !request.request || !compilerOptions.paths) {
+  if (!request || !request.request) {
```

Comment:
Note to reviewer, why should module resolution for typescript files be different from the standard tsc compiler resolution strategy.

Reply:
Typescript resolves packages for either TypeScript source or type definitions.  When bundling, the actual executable javascript for the given package is required.